lib/MANIFEST.in

```diff
@@ -1,4 +1,4 @@
-include README.md
```

Comment:
This line isn't actually doing anything because our `README.md` file lives in the root directory of the repo, not in `lib/`